ibm/service/catalogmanagement/resource_ibm_cm_validation.go

```diff
@@ -242,7 +242,7 @@ func resourceIBMCmValidationCreate(context context.Context, d *schema.ResourceDa
 		validateInstallOptions.SetEnvironmentVariables(envsModel)
 	}
 	if _, ok := d.GetOk("schematics"); ok {
-		schematicsModel, err := schematicsMapToDeployRequestBodySchematics(d.Get("schematics").(map[string]interface{}))
+		schematicsModel, err := schematicsMapToDeployRequestBodySchematics(d.Get("schematics.0").(map[string]interface{}))
```

Comment:
"schematics" is  optional can we have some length check before we access 0 index

Reply:
pushed change to check d.GetOk("schematics.0") before accessing it.